src/buffer/out/textBuffer.hpp

```diff
@@ -135,6 +135,8 @@ class TextBuffer final
     bool MoveToNextWord(COORD& pos, const std::wstring_view wordDelimiters, COORD lastCharPos) const;
     bool MoveToPreviousWord(COORD& pos, const std::wstring_view wordDelimiters) const;
 
+    const std::vector<SMALL_RECT> GetTextRects(COORD start, COORD end, bool blockSelection = false) const;

```

Comment:
Please ensure this gets a unit test in the text buffer unit tests now that it lives over here.

Reply:
Added. Note: I'm going a little light on them on purpose because this same stuff gets tested in selection tests and UIA tests.